dartsim/src/JointFeatures.cc

```diff
@@ -161,6 +161,7 @@ void JointFeatures::DetachJoint(const Identity &_jointId)
   freeJoint->setSpatialVelocity(spatialVelocity,
           dart::dynamics::Frame::World(),
           dart::dynamics::Frame::World());
+  child->incrementVersion();
```

Comment:
Do you want to leave these calls in after DART has been patched? If not, maybe add a `/todo`?

Reply:
I added a todo in 4458614